From cfe1b9562dd6db092f05406a4cdbc805c85fe828 Mon Sep 17 00:00:00 2001
From: Peter Lohmer <peterlohmer01@gmail.com>
Date: Wed, 21 Nov 2018 14:40:32 +0100
Subject: [PATCH 7/7] wmglobal.h: set new version number and added comments
 what's new

Signed-off-by: Peter Lohmer <peterlohmer01@gmail.com>
---
 wmglobal.h |   10 ++++++++--
 1 Datei geändert, 8 Zeilen hinzugefügt(+), 2 Zeilen entfernt(-)

diff --git a/wmglobal.h b/wmglobal.h
index 1b3a434..bea02dd 100644
--- a/wmglobal.h
+++ b/wmglobal.h
@@ -94,8 +94,14 @@
 //                  erweitert. kontext menu ergänzt für die nachkommastellenzahl von rcf zu editieren.
 // v2.29 11.07.2018 bei der frequenzberechnung den sollwert im falle von dc betrieb fest auf 0 gesetzt. hier muss später
 //                  hubertus nochmal ran, weil wir sonst im falle eines signales mit f > 0 trotzdem 0 anzeigen.
-
-#define WMVersion "V2.29"
+// v2.30 31.07.2018 hardware misst jetzt frequenz bis runter zu 5.96 hz. dies ist jetzt richtig berücksichtigt.
+// v2.31 21.11.2018 dc messwerte werden jetzt mit vorzeichen im interface gesendet. der ac button im konfigurations menu ist
+//                  disabled wenn wm3000 nicht mit dc option gestartet wurde. es wurde eine funktion eingebaut um die justage
+//                  daten version abzufragen. hintergrund ist dass geräte die noch einen alten justage daten stand haben
+//                  (also vor dc) trotzdem als justiert gelten solange die software ohne dc option gestartet wird.
+//                  es wurden neue übersetzungs files für englisch und polnisch hinzugefügt.
+
+#define WMVersion "V2.31"
 
 #define wm3000uHome QDir::homePath()
 // #define ServerCommLogFilePath "/usr/share/wm3000u/log/ServerComm.log"
-- 
1.7.10.4

